src/unstract/adapters/__init__.py

```diff
@@ -1,7 +1,14 @@
+__version__ = "0.4.0"
+
 import logging
 from logging import NullHandler
 from typing import Any
 
 logging.getLogger(__name__).addHandler(NullHandler())
 
 AdapterDict = dict[str, dict[str, Any]]
+
+
+def get_adapter_version() -> str:
```

Comment:
Just one question @chandrasekharan-zipstack How does the system know that it needs to call this particular function as we are not referring to it anywhere? We only mention the filename in __init__.py

Reply:
Or this is just an enabler I guess? Just noticed line 1 __version__ = "0.4.0" Guess that is the one doing the trick?

Reply:
Yeah @gaya3-zipstack - its handled by [PDM](https://backend.pdm-project.org/metadata/#dynamic-project-version)